app/views/landing-pages/league/components/Leaderboard.vue

```diff
@@ -61,6 +59,33 @@ export default {
     getCountryName (row) {
       return utils.countryCodeToName(row.creatorCountryCode)
     }
+  },
+
+  computed: {
+    ...mapGetters({
```

Comment:
Would it be more or less expensive to send hundreds of students into each leaderboard component as props? My gut feeling is that this is better, because it relies directly on Vuex. 

A potential filtering could be useful at some stage, but it is not clear where we'd want to do it yet, given the global leaderboard, and that any of the teacher's students could be visible there.

Reply:
Great call out!
I have some opinions on the data flow that may resolve this issue -- will put on server PR.